### PR TITLE
fix app upgrade show-versions option

### PR DESCRIPTION
**Pull request by gitlawr**
Address issue: https://github.com/rancher/rancher/issues/16478

### CODE REVIEW

**Comment by orangedeng** [comment]
LGTM

**Comment by StrongMonkey** [comment]
LGTM

File: cmd/app.go [hunk 1]
```diff
@@ -290,7 +290,7 @@ func appUpgrade(ctx *cli.Context) error {
 		return err
 	}
 
-	if ctx.Bool("versions") {
+	if ctx.Bool("show-versions") {
```
Comment (alena1108) [inline cmd/app.go]: @dramich just wanted to confirm that the intended param name as we have it `versions` in one place, and `show-versions` in another. As it never worked anyway, we have flexibility to decide on the name now

Comment (dramich) [reply cmd/app.go]: @alena1108 I think leaving the flag as `show-versions` is fine, but I am not stuck on it. If you want to change it to `versions` that's fine as well.

**Pull request merged**